packages/core/src/utils/write-file.ts

```diff
@@ -32,3 +35,16 @@ export const writeIfRequired = async ({
     await writeFile({ dir, fileName, contents });
   }
 };
+
+export const writePackageJson = async <T = PackageJson>({
```

Comment:
Might be worth adding a `T extends PackageJson` here to prevent strings being passed in.